src/components/purchase-complete/index.js

```diff
@@ -62,12 +62,46 @@ const PurchaseComplete = ({
 
     if (!checkout) return null;
 
-    let footerHasTicketText = (rest.hasOwnProperty('footerHasTicketText') && typeof rest.footerHasTicketText !== 'undefined' ?
-        rest.footerHasTicketText : T.translate('purchase_complete_step.footer_has_ticket_text'));
+    let orderComplete1stParagraph = (
+        currentUserTicket ? 
```

Comment:
if currentUserTicket == true , then its means that its the initial order, so, should be using here
if defined , initialOrderComplete1stParagraph otherwise
https://github.com/fntechgit/summit-registration-lite/blob/2d8a39f6f09729c69131c526732ccc6bff598fcc/src/i18n/en.json#L4